stories/s1grdiw.eocloud.stories.js

```diff
@@ -335,17 +316,13 @@ export const findDatesEPSG4326 = () => {
 };
 
 export const findDatesEPSG3857 = () => {
-  const layer = new S1GRDEOCloudLayer(
+  const layer = new S1GRDEOCloudLayer({
     instanceId,
     layerId,
-    null,
-    null,
-    null,
-    null,
-    AcquisitionMode.IW,
-    Polarization.DV,
-    OrbitDirection.ASCENDING,
-  );
+    acquisitionMode: AcquisitionMode.IW,
+    polarization: Polarization.DV,
+    resolution: OrbitDirection.ASCENDING,
```

Comment:
repeated just for easy find: resolution isn't a parameter for S1GRDEOCloudLayer anymore

also same mistake as above (resolution: OrbitDirection.ASCENDING,)